test_app/lib/screens/join_screen.dart

```diff
@@ -67,8 +67,8 @@ class _JoinConferenceContentState extends State<JoinConferenceContent> {
   final _dolbyioCommsSdkFlutterPlugin = DolbyioCommsSdk.instance;
   final formKeyAlias = GlobalKey<FormState>();
   final formKeyId = GlobalKey<FormState>();
-  bool isJoining = false;
-  bool isReplaying = false;
+  bool isJoinButtonPressed = false;
+  bool isReplayButtonPressed = false;
```

Comment:
Maybe `joiningInProgress` and `startingConfrenceReplay`. Any suggestions are welcome :-)

Reply:
I think your suggestions are more appropriate than the current name, I'll apply it :)